client-tez/src/test/java/org/apache/tez/dag/app/TezRemoteShuffleManagerTest.java

```diff
@@ -224,7 +227,12 @@ public void testTezRemoteShuffleManagerSecure() {
       secretManager.addTokenForJob(tokenIdentifier, sessionToken);
       TezRemoteShuffleManager tezRemoteShuffleManager =
           new TezRemoteShuffleManager(
-              appId.toString(), sessionToken, conf, appId.toString(), client, null);
+              appId.toString(),
+              sessionToken,
+              new TezClientConf(conf),
```

Comment:
I think there needs a way to init and read configuration, like:

```
  public ShuffleServerConf(String fileName) {
    super();
    boolean ret = loadConfFromFile(fileName);
    if (!ret) {
      throw new IllegalStateException("Fail to load config file " + fileName);
    }
  }
```


```
  public DashboardConf(String fileName) {
    super();
    boolean ret = loadConfFromFile(fileName);
    if (!ret) {
      throw new IllegalStateException("Fail to load config file " + fileName);
    }
  }
```

```
  public CoordinatorConf(String fileName) {
    super();
    boolean ret = loadConfFromFile(fileName);
    if (!ret) {
      throw new IllegalStateException("Fail to load config file " + fileName);
    }
  }
```

Reply:
I means that new TezClientConf() without any parameter, then set TEZ_RSS_TEST_MODE_ENABLE. 
Then TezClientConf without any parameter will be zero cost. 
Here we must load configuration, I think it is not neccessary. Because the configuration in TezClientConf(new Configuration) doesn't pass any useful config.

Reply:
I think set TEZ_RSS_TEST_MODE_ENABLE should be obvious, and not  like hidden in TezClientConf().
The normal case should read config from client side by Configuration, and convert to TezClientConfig, so there need TezClientConfig(new Configuration()).
And no need new TezClientConfig(), because if so, there will be default config and meanless.

So, I changed the test case to be:TezClientConf(new Configuration()), new Configuration() just like conf from client side.